public/smokeAlarmPortal_files/base.css

```diff
@@ -347,4 +347,10 @@ a.contrast-link {
     background-color: rgba(0, 0, 0, 0.3);
     -moz-border-radius: 5px 5px 5px 5px;
     border-radius: 5px 5px 5px 5px; /* standards-compliant: (IE) */
-}
\ No newline at end of file
+}
+
+.img-square {
```

Comment:
This seems like an overly generic CSS class name for an image that we know is specifically meant to appear on a 311-route page to the right of the usual Red Cross logo.  Sure, we might add future 311 cities and they would have their own routes, but the CSS for this image is still 311-specific (as well as perhaps being square-specific).  Could the class have a name that indicates its function more closely?